crates/bevy_reflect/src/type_registry.rs

```diff
@@ -160,13 +168,15 @@ impl TypeRegistryArc {
 /// [`TypeId`]: std::any::TypeId
 /// [name]: std::any::type_name
 /// [short name]: TypeRegistration::get_short_name
+/// [`TypeInfo`]: crate::TypeInfo
 /// [0]: crate::Reflect
 /// [1]: crate::Reflect
 pub struct TypeRegistration {
     type_id: TypeId,
     short_name: String,
     name: &'static str,
```

Comment:
The name and type_id fields are now redundant with `type_info().id()`

Reply:
Good point, I'll fix that. 

I think I'll keep `short_name` in here, though. I don't know if it's worth storing on `TypeIdentity` directly.